docs/Tooling/stacked-cli.md

```diff
@@ -81,7 +81,7 @@ And underneath your last route add
 // @stacked-route
 ```
 
-Now if you run `stacked create view myView` you'll see that all the files are generated AND we also add the route into your `app.dart` file. The modifications are optional so if you don't have it they won't happen and everything else will still work. 
```

Comment:
This is not bad, I changed it only to avoid confusion because the result of that view will be `MyViewView`.

Reply:
Good choice